packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -747,9 +750,53 @@ contract Deploy is Deployer {
                 splitDepth: cfg.faultGameSplitDepth(),
                 clockExtension: Duration.wrap(uint64(cfg.faultGameClockExtension())),
                 maxClockDuration: Duration.wrap(uint64(cfg.faultGameMaxClockDuration())),
-                vm: IBigStepper(
-                    new AlphabetVM(outputAbsolutePrestate, IPreimageOracle(artifacts.mustGetAddress("PreimageOracle")))
-                ),
+                vm: IBigStepper(artifacts.mustGetAddress("MipsSingleton")),
+                weth: weth,
+                anchorStateRegistry: IAnchorStateRegistry(artifacts.mustGetAddress("AnchorStateRegistryProxy")),
+                l2ChainId: cfg.l2ChainID()
+            })
+        });
+    }
+
+    /// @notice Sets the implementation for the `PERMISSIONED_SUPER_CANNON` game type in the `DisputeGameFactory`
+    function setSuperPermissionedGameImplementation() public onlyDevnet broadcast {
+        console.log("Setting SuperPermissionedDisputeGame implementation");
+        IDisputeGameFactory factory = IDisputeGameFactory(artifacts.mustGetAddress("DisputeGameFactoryProxy"));
+        IDelayedWETH weth = IDelayedWETH(artifacts.mustGetAddress("DelayedWETHProxy"));
+
+        _setFaultGameImplementation({
+            _factory: factory,
+            _params: IFaultDisputeGame.GameConstructorParams({
+                gameType: GameTypes.SUPER_CANNON,
+                absolutePrestate: loadMipsAbsolutePrestate(),
```

Comment:
For the super game types, the interop prestate must be used rather than the pre-interop prestate.